pyro/contrib/tracking/hashing.py

```diff
@@ -31,12 +31,12 @@ class LSH(object):
         >>> lsh.add('b', b)
         >>> lsh.add('c', c)
         >>> lsh.nearby('a') # even though c is within 2radius of a
-        set(['b'])
+        {'b'}
```

Comment:
I don't think this will work in python 2.7..maybe its best to to skip the output check for these -  # doctest: +SKIP.